lib/datasource/docker/index.ts

```diff
@@ -113,10 +113,12 @@ async function getECRAuthToken(
   region: string,
   opts: HostRule
 ): Promise<string | null> {
-  const config = { region, accessKeyId: undefined, secretAccessKey: undefined };
+  const config = { region, credentials: undefined };
```

Comment:
Is there a type in afk to use instead of adding untyped here?

Reply:
Yes, good hint, see 65c26fc3c5f2a34cc3b1123c048f6c74fd3fa08b).